src/rf/apps/workers/process.py

```diff
@@ -81,6 +83,8 @@ def handle_message(self, record):
             return self.thumbnail(record)
         elif job_type == JOB_HANDOFF:
             return self.emr_hand_off(record)
+        elif job_type == JOB_HEARTBEAT:
+            return self.emr_heartbeat(record)
```

Comment:
After discussion, the answer is "yes".

The heartbeat allows us to fail-fast if the cluster has terminated before chunk/mosaic completion, and the existing timeout allows us to mark layers as failed if the cluster runs continuously without completing or failing for a long time (something we have experienced more than once).